packages/gatsby-plugin-google-analytics/README.md

```diff
@@ -79,3 +79,21 @@ If you enable this optional option, Google Analytics will not be loaded at all f
 ## The "exclude" option
 
 If you need to exclude any path from the tracking system, you can add it (one or more) to this optional array as glob expressions.
+
+## Create Only Fields
```

Comment:
Can you add an example here of how you'd add some of these to your gatsby-config.js?

Reply:
Sure, see d4d27c3c18aeab3fb8f74cdd6e79af379e57b95e